elements/pfe-absolute-position/src/pfe-absolute-position.js

```diff
@@ -210,10 +210,21 @@ class PfeAbsolutePosition extends PFElement {
    * @returns {void}
    */
   updatePosition() {
+    // if we are currently updating the position then ignore this request
+    if (this._updatingPosition) return;
     if (this.__observe === true) {
+      // set the internal state to updating.
+      this._updatingPosition = true;
+      // ask the manager to update our position over the target element.
       this.__manager.positionElement(this);
+      // we are only going to debounce this for one second.
```

Comment:
@heyMP - Not quite one second. Can you change this to 100 ms?

Reply:
👍